src/js/ToolTip.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './ToolTip.css';
+
+const ToolTip = props => {
+  const { hoverLoc, activePoint } = props;
+  const svgLocation = document
+    .getElementsByClassName('linechart')[0]
+    .getBoundingClientRect();
+
+  const placementStyles = {};
+  const width = 115;
+  placementStyles.width = width + 'px';
+  placementStyles.left = hoverLoc + svgLocation.left - width / 2;
+
+  return (
+    <div className="hover" style={placementStyles}>
+      <div className="date">{activePoint.d}</div>
+      <div className="price">{activePoint.p}</div>
+    </div>
+  );
+};
+
+// ToolTip.propTypes = {
```

Comment:
Commented out code 🙄 

Though I do approve of putting prop types in the project.